tests/tui/test_organization_preview_coverage.py

```
@@ -203,6 +203,155 @@ def test_action_refresh_preview(self) -> None:
+
+        view.action_confirm()
+
+        assert mock_panel.update.call_args_list == [
+            call("[dim]Applying organization...[/dim]"),
+            call("[dim]Working...[/dim]"),
+        ]
+        view._set_status.assert_called_once_with("Applying organization...")
+        view._apply_organization.assert_called_once()
+
+    def test_apply_organization_success_uses_real_organizer(self) -> None:
+        view = OrganizationPreviewView(input_dir="/tmp/in", output_dir="/tmp/out")
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/in"

(S108)